pkgs/os-specific/linux/nvidia-x11/default.nix

```diff
@@ -117,6 +117,14 @@ rec {
     persistencedSha256 = "sha256-NuqUQbVt80gYTXgIcu0crAORfsj9BCRooyH3Gp1y1ns=";
 
     broken = kernel.kernelAtLeast "6.2";
+
+    # fixes the bug described in https://bbs.archlinux.org/viewtopic.php?pid=2083439#p2083439
+    # see https://bbs.archlinux.org/viewtopic.php?pid=2083651#p2083651
+    # and https://bbs.archlinux.org/viewtopic.php?pid=2083699#p2083699
+    postInstall = ''
```

Comment:
Shouldn't `$lib32/lib/tls` also be affected by this problem?

Reply:
I don't know ...

Reply:
Did some digging, it looks like there are technically versions where this is an issue, but none of the ones we package have it.

* double tls libraries were removed in `415.13` specifically because debian sometimes disabled the hwcap overrides feature, essentially running into this exact issue.
* lib32 was added in driver `391+`, so `340` and `390` arent affected.